app.js

```diff
@@ -0,0 +1,194 @@
+
+// // Easy Going
+ for( let q = 1; q <= 20; q = q + 1) {
+ 	console.log(q);
+ }
+
+// // Get Even
+ for( let q = 0; q <= 200; q++){
+ 	if((q % 2) === 0) { 
+ 		console.log(q)
+
+ 	}
+}
+
+
+// // Excited Kitten
+
+
+for(let i = 1; i < 20; i++) {
+	const meowArr = ["...human...why you taking pictures of me?...", "...the catnip made me do it...","...why does the red dot always get away..."];
+	const randomValue = meowArr[Math.floor(meowArr.length * Math.random())];
+
+	
+ if( i % 2 === 0) { 
+
+ 	console.log(randomValue);
+
+ 	} else { 
+	console.log("Love me, pet me! HSSSSSS!")
+ }
+}
+
+
+// // Fizz Buzz
+// this took me a little bit and I used
+// google and the help of others
+// but the more I let it play through 
+// the more it makes sense
+for( let i = 1; i<=100; i++) {
+	if( i % 15 === 0)
+		console.log("fizzbuzz");
+
+else if( i % 3 === 0)
+	console.log("fizz");
+else if ( i % 5===0)
+	console.log("buzz");
+else console.log(i);
+
+}
```

Comment:
nice job solving the problem.  now, you've got to fix your indentation! I'd also hold off on omitting curly braces just yet.
```
for( let i = 1; i<=100; i++) {
    if( i % 15 === 0) {
        console.log("fizzbuzz");
    }
    else if( i % 3 === 0) {
        console.log("fizz");
    }
    else if ( i % 5===0) {
        console.log("buzz");
    }
    else {
        console.log(i);
    }
}
```